scrapegraphai/graphs/abstract_graph.py

```diff
@@ -221,15 +204,18 @@ def handle_model(model_name, provider, token_key, default_token=8192):
                 raise KeyError("Model not supported") from exc
             return OneApi(llm_params)
 
-        if "nvidia" in llm_params["model"]:
+        elif "nvidia" in llm_params["model"]:
+
             try:
                 self.model_token = models_tokens["nvidia"][llm_params["model"].split("/")[-1]]
                 llm_params["model"] = "/".join(llm_params["model"].split("/")[1:])
             except KeyError as exc:
                 raise KeyError("Model not supported") from exc
             return ChatNVIDIA(llm_params)
+        else:
+            model_name = llm_params["model"].split("/")[-1]
+            return handle_model(model_name, llm_params["model"], model_name)
 
-        # Raise an error if the model did not match any of the previous cases
         raise ValueError("Model provided by the configuration not supported")
```

Comment:
Unreachable code, because the `if/elif/else` block causes it to be skipped in all cases. The old version constisted only of `if` statements because there would be no fall-through in any case